### ISSUE TITLE
Force python version for dstat and other tools

### ISSUE THREAD

**Issue by hechi** [peelframework/peel#111]
Hey, 

i enabled dstat-0.7.2 and start my experiments. But there was no output of dstat.
I found out that dstat-0.7.2 printed an error which is piped to /dev/null on the ssh cli invocation. 
The error was a python syntax error which appears between python2.y and python3.x scripts.
```
/../dstat-0.7.2/dstat", line 119
    except getopt.error, exc:
```
Probably it would be a good idea for the peelframework to force specific python versions or print the dstat error messages. Just a suggestion :)

**Comment by aalexandrov** [comment]
I guess the workaround would be to call dstat indirectly

```bash
/usr/bin/python2 /usr/bin/dstat
```

instead of 

```bash
/usr/bin/dstat
```

**Comment by hechi** [comment]
Yes as a workaround it would work, but not every python environment has to be in /usr/bin/ :)
probably it would be enough if you but the error pipe into the log and not to /dev/null
thx for helping :)
